File: 1 ano/Log de Programação/2°Bi/Lista 2/CasaPy/Lista2-casa_Exercício 7 - Curso.py
codigo = int(input("Insira o código de acesso do seu curso: "))
match codigo:
    case 1:
        print("Engenharia")
    case 2:
        print("Edificações")
    case 3:
        print("Sistemas elétricos")
    case 4:
        print("Turismo")
    case 5:
        print("Análise de Sistemas")
    case _:
        print("Código Inválido")

File: 1 ano/Log de Programação/2°Bi/Lista 2/CasaC/Lista2-casa_Exercício 7 - Curso.c
#include<stdio.h>

int main()
{
    int codigo;

    printf("Insira o codigo de acesso de seu curso: ");
    scanf("%i", &codigo);

    switch (codigo){
        case 1:
            printf("Engenharia");
            break;
        case 2:
            printf("Edificacoes");
            break;
        case 3:
            printf("Sistemas Eletricos");
            break;
        case 4:
            printf("Turismo");
            break;
        case 5:
            printf("Analise de Sistemas");
            break;
        default:
            printf("Codigo Invalido");
            break;

    }
}